step-definitions/utils/ui.js

```diff
@@ -179,27 +178,20 @@ const reallocateCaseToForce = async function (force) {
 
 const canSeeContentInTable = async function (value) {
   let newValue = value;
-  if (value === "(Submitted)") {
+  if (value === "(Submitted)" || value === "Resolved") {
     newValue = newValue.replace(/[()]/g, "").toUpperCase();
   }
-  newValue = newValue.replace(/^PR(\d+)/, "TRPR00$1").replace(/^PS(\d+)/, "TRPS00$1"); // TODO: remove this once we update new UI to display PR0* instead of full trigger code
-  const found = await reloadUntilContentInSelector(
-    this.browser.page,
-    newValue,
-    "#main-content > div.moj-filter-layout > div.moj-filter-layout__content > div.moj-scrollable-pane > div > table > tbody"
-  );
+
+  newValue = value.replace(/^PR(\d+)/, "TRPR00$1").replace(/^PS(\d+)/, "TRPS00$1"); // TODO: remove this once we update new UI to display PR0* instead of full trigger code
```

Comment:
Is there a reason for this `newValue = value.replace`? It used to be `newValue = newValue.replace`

It wipes out line 182.